app/services/github/contributions/engineers_wrapper.rb

```diff
@@ -0,0 +1,38 @@
+module Github
+  module Contributions
+    class EngineersWrapper
+      def initialize(company:)
+        @company = company
+      end
+
+      def get_engineer_id_by_github_user(github_user)
+        engineers.to_h[github_user]
+      end
+
+      def to_query
+        engineers.map do |github_user, user_id|
+          "author:#{github_user}"
+        end.join(' ')
+      end
+
+      def has_engineers?
+        not engineers.empty?
```

Comment:
if possible use assertive methods, to improve the readability 😄 
```suggestion
        engineers.any?
```

Reply:
sure!